platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/internal/reconnect/ReconnectHashListenerTest.java

```diff
@@ -164,6 +164,22 @@ public void close() throws IOException {
             delegate.close();
         }
 
+        @Override
+        public void saveRecords(
+                long firstLeafPath,
+                long lastLeafPath,
+                Stream<VirtualHashRecord> pathHashRecordsToUpdate,
+                Stream<VirtualLeafRecord<TestKey, TestValue>> leafRecordsToAddOrUpdate,
+                Stream<VirtualLeafRecord<TestKey, TestValue>> leafRecordsToDelete,
+                boolean isReconnectContext)
+                throws IOException {
+            final var ir = pathHashRecordsToUpdate.toList();
+            this.internalRecords.add(ir);
+            final var lr = leafRecordsToAddOrUpdate.toList();
+            this.leafRecords.add(lr);
+            delegate.saveRecords(firstLeafPath, lastLeafPath, ir.stream(), lr.stream(), leafRecordsToDelete);
```

Comment:
Should this test use the new method with "true" as the last arg?

Reply:
Yes. It should be. Fixed.